docs/project-service-management-guide.md

```diff
@@ -156,106 +152,101 @@ deploy all manifests in this repository to the cluster in a set interval.
 - [Clone the repository.](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-new-repo?view=azure-devops#clone-the-repo-to-your-computer)
 - Initialize the project via `spk`
   ```
-  spk project init
-  git add -A
-  git commit -m "Initializing application repository."
-  git push -u origin --all
+  $ spk project init
+  $ git add -A
+  $ git commit -m "Initializing application repository."
+  $ git push -u origin --all
   ```
-- Create Variable Group via `spk` (optional flag parameters can be used if `spk`
-  was not intialized)
+- Create a variable group via `spk`. If you are using a repo per service source
+  control strategy, you only need to do this once.
   ```
-  VARIABLE_GROUP_NAME=<my-vg-name>
-  spk project create-variable-group $VARIABLE_GROUP_NAME -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
-  git add -A
-  git commit -m "Adding Project Variable Group."
-  git push -u origin --all
+  $ export VARIABLE_GROUP_NAME=<my-vg-name>
+  $ spk project create-variable-group $VARIABLE_GROUP_NAME -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
+  $ git add -A
+  $ git commit -m "Adding Project Variable Group."
+  $ git push -u origin --all
   ```
   where `ACR_NAME` is the name of the Azure Container Registry where the Docker
   Images will be served from and `SP_APP_ID`, `SP_PASS`, and, `SP_TENANT` are an
   associated Service Principal's ID, Password, and Tenant, that have Read and
   Write access to the ACR.
 - Deploy the lifecycle pipeline (optional flag parameters can be used if `spk`
   was not intialized)
+
   ```
-  spk project install-lifecycle-pipeline
+  $ spk project install-lifecycle-pipeline --org-name $ORG_NAME --devops-project $DEVOPS_PROJECT --repo-url $SERVICE_REPO_URL --repo-name $SERVICE_NAME --pipeline-name $SERVICE_NAME-lifecycle-pipeline
   ```
 
+  where `ORG_NAME` is the name of your Azure Devops org, `DEVOPS_PROJECT` is the
+  name of your Azure Devops project, `SERVICE_REPO_URL` is the url that you used
+  to clone your service from Azure Devops, and `SERVICE_NAME` is the name of the
+  service.
+
+  Note: If you are using a repo per service source control strategy you should
+  run `install-lifecycle-pipeline` once for each repo.
+
 **NOTE** `spk project` command documentation can be found
 [here](/docs/project-management.md).
 
-##### Adding a Service to a Application Repository
+#### Adding a Service to a Application Repository
 
 - [Clone the repository.](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-new-repo?view=azure-devops#clone-the-repo-to-your-computer)
 - Create the service via `spk`, there are optional parameters that _should_ be
   used to configure the service and its associated helm charts (other optional
   flag parameters can be used if `spk` was not intialized)
   ```
   SERVICE_NAME=<my-new-service-name>
-  spk service create $SERVICE_NAME
+  spk service create . --display-name $SERVICE_NAME ...
```

Comment:
I kind of want to steer users away from using the `.` notation for placing their services under the root directory, or perhaps we can have explicit instructions for placing a service in the project root vs. placing the service in a subdirectory.

Do we envision more single application repositories or "monorepos"?

Hopefully our refactoring will make the behaviors clear and consistent soon.

Reply:
I think we will have a mix of both monorepos and single repo per service so we should describe both.